src/main/java/edu/hawaii/its/groupings/controller/ErrorRestController.java

```diff
@@ -29,4 +32,12 @@ public ResponseEntity<Void> feedbackError(@RequestBody Map<String, String> body,
         session.setAttribute("feedback", new Feedback(exceptionMessage));
         return new ResponseEntity<>(HttpStatus.NO_CONTENT);
     }
+
+    @PreAuthorize("hasRole('ADMIN')")
+    @GetMapping(value = "/api/groupings/testing/exception")
+    public ResponseEntity<String> throwException() {
+        logger.info("Entered REST throwException...");
+
+        throw new ExceptionForTesting("ThrowException");
```

Comment:
Could you set the message as "Exception thrown intentionally" so that the message is clear for someone who might be confused what this exception is? Thanks.